Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Added
 - Added support of F13-F24 keys. [UUM-44328](https://issuetracker.unity3d.com/product/unity/issues/guid/UUM-44328)
+- Added missing manuals for the TrackedPoseDriver and TrackedDeviceRaycaster components. [ISXB-1410](https://jira.unity3d.com/browse/ISXB-1410)
```

Comment:
Why do we link to private Jira URLs in the changelog? Users can't access non-UUM tickets, I believe. `jira.unity3d.com` is credentialed. 

```suggestion
- Added missing documentation for the Tracked Pose Driver and Tracked Device Raycaster components. [ISXB-1410](https://jira.unity3d.com/browse/ISXB-1410)
```

Reply:
```suggestion
- Added missing documentation for the Tracked Pose Driver and Tracked Device Raycaster components. [ISXB-1410](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1410)
```
This link should work: https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1410

Reply:
That works 👍